packages/connect-examples/expo-playground/app/routes/_index.tsx

```
@@ -184,9 +195,21 @@ export default function IndexPage() {
 
                     {/* 设备信息 - 剩余空间填充 */}
                     <div className="flex-1">
-                      <h3 className="font-semibold text-lg mb-4 text-foreground">
-                        {t('common.deviceInfo')}
-                      </h3>
+                      <div className="flex items-center justify-between mb-4">
+                        <h3 className="font-semibold text-lg text-foreground">
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)